Mathlib/AlgebraicTopology/ModelCategory/CategoryWithCofibrations.lean

```diff
@@ -0,0 +1,142 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.MorphismProperty.Basic
+
+/-!
+# Categories with classes of fibrations, cofibrations, weak equivalences
```

Comment:
What's the reasoning for choosing cofibrations in the file name?

Reply:
I could have used `CategoryWithFibrations`, but this could create a confusion with the established notion of "category of fibrant objects" in the literature.